apm-sniffer/apm-sdk-plugin/dubbo-2.7.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/asf/dubbo/DubboInterceptor.java

```diff
@@ -152,4 +165,26 @@ private String generateRequestURL(URL url, Invocation invocation) {
         requestURL.append(generateOperationName(url, invocation));
         return requestURL.toString();
     }
+
+    private void collectArguments(boolean needCollectArguments, int argumentsLengthThreshold, AbstractSpan span, Invocation invocation) {
+        if (needCollectArguments && argumentsLengthThreshold > 0) {
+            Object[] parameters = invocation.getArguments();
+            if (parameters != null && parameters.length > 0) {
+                StringBuilder stringBuilder = new StringBuilder();
+                boolean first = true;
+                for (Object parameter : parameters) {
+                    if (!first) {
+                        stringBuilder.append(",");
+                    }
+                    stringBuilder.append(parameter);
```

Comment:
So, this depends on `parameter#toString`, right? Is this a recommended way? Or should we consider JSON? I am not using Dubbo for a long time. Is Dubbo having toString already?

Reply:
It's the object's own toString, which the user has to implement, I had modify markdown file